src/utilities/effects.ts

```diff
@@ -5583,31 +5582,28 @@ const effects = {
 
   async uploadDictionary(
     dictionary: string,
-    type: DictionaryTypes,
     user: User | null,
-  ): Promise<CommandDictionary | ChannelDictionary | ParameterDictionary | null> {
-    const typeString = type.charAt(0).toUpperCase() + type.slice(1);
-
+  ): Promise<{ channel?: ChannelDictionary; command?: CommandDictionary; parameter?: ParameterDictionary } | null> {
```

Comment:
Does removing the type mean that one dictionary file can now contain channel, command, and parameter dictionaries at the same time?
If so, is this still backwards compatible with the old format?

Reply:
Yes, it does, FPrime dictionaries appear to contain all 3. This is backward compatible with what we had as Clipper uploads 1 dictionary type at a time. The dictionary parsers on the server is responsible for determining which type of dictionary is uploaded and parsed correctly. 

AMPCS dictionary parser:
https://github.com/NASA-AMMOS/aerie/pull/1542/commits/ad42bf616dcc58e2c8c9e58ba630cf68b9286f9c

FPrime dictionary parser:
https://github.com/NASA-AMMOS/aerie/pull/1542/commits/9821ebce21538ddb819106de4579ebfd0d7a5445